src/ProductFeedStatus.php

```diff
@@ -54,8 +45,7 @@ class ProductFeedStatus {
 	 */
 	public static function get() {
 
-		$local_feed  = self::get_local_feed();
-		$data_prefix = PINTEREST_FOR_WOOCOMMERCE_PREFIX . '_feed_' . $local_feed['feed_id'] . '_';
+		$data_prefix = PINTEREST_FOR_WOOCOMMERCE_PREFIX . '_feeds_';
```

Comment:
We repeat this often, so it might make more sense to have its own constant in this class.

Reply:
Yes.